mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -19,6 +19,7 @@
 
     <!-- "Snackbar" shown after the user has pressed the 'erase' button at the end of a browsing session -->
     <string name="feedback_erase">Your browsing history has been erased.</string>
+    <string name="feedback_erase2">Browsing history cleared</string>
```

Comment:
@ionutbedregeanu: is this new string supposed to replace the one above? or is it an additional string? thanks

Reply:
Hi! For now is a mvp additional string. I should rename it?

Reply:
Totally fine, just wanted to double check. Thank you for confirming!